input/pagecontent/3-2-2-Heartbeat.md

```diff
@@ -1,16 +1,16 @@
-{% include questionnote.html text='Implementer feedback is requested for the need to support heartbeats for WebSockets.' %}
+{% include questionnote.html text='Implementer feedback is requested for the need to support Heartbeat.htmls for WebSockets.' %}

```

Comment:
Do you mean to use Heartbeat.htmls or Heartbeat events?  My guess is a copy/paste thing assuming that it was going to be in a link.  Couple of places in this file and other infrastructure events.